server/athenian/api/align/queries/metrics.py

```diff
@@ -171,49 +209,89 @@ async def _calculate_team_metrics(
     calculator = make_calculator(account, meta_ids, mdb, pdb, rdb, cache)
     tasks = []
     if pr_metrics:
-        pr_participants = [{
-            PRParticipationKind.AUTHOR: team,
-        } for team in _loginify_teams(teams, prefixer)]
-        tasks.append(calculator.calc_pull_request_metrics_line_github(
-            pr_metrics, time_intervals, quantiles, [], [], [repos], pr_participants,
```

Comment:
forbid multiple parameters on the same line, if the parameter list is on multiple lines